dotcom-rendering/src/footballTables.ts

```diff
@@ -0,0 +1,33 @@
+import type { TeamScore } from './footballMatches';
+
+export type TeamResult = {
+	id: string;
```

Comment:
Do we know what information in the data coming from frontend we'll be able to use for this `id`?

Reply:
We have found in frontend that there is a match ID associated with the scores. I've made an issue to track this
https://github.com/guardian/dotcom-rendering/issues/13690